cmd/bbolt/command_surgery_cobra.go

```diff
@@ -78,3 +78,39 @@ func surgeryClearPageElementFunc(cmd *cobra.Command, args []string) error {
 	fmt.Fprintf(os.Stdout, "All elements in [%d, %d) in page %d were cleared\n", surgeryStartElementIdx, surgeryEndElementIdx, surgeryPageId)
 	return nil
 }
+
+func newSurgeryAbandonFreelistCommand() *cobra.Command {
+	abandonFreelistCmd := &cobra.Command{
+		Use:   "abandon-freelist <bbolt-file> [options]",
```

Comment:
Maybe we prefer to be more structured: 
```bbolt surgery freelist abandon ....```
this way we could add in a future: 
```bbolt surgery freelist rebuild/check ....```

Reply:
Great point!

Reply:
Done. PTAL, thx